src/main.rs

```diff
@@ -786,6 +790,22 @@ fn try_main() -> Result<()> {
                 )?;
                 println!("Metadata file created at {path:?}");
             }
+            NonmemCommands::Sitrep { model_path } => {
+                if !model_path.exists() {
+                    bail!("Model file does not exist: {model_path:?}");
+                }
+                let (config_path, nonmem_config) = load_nonmem_config(None)?;
+                println!("Config loaded from {config_path:?}");
+                let mpi_version = nonmem_config.validate()?;
```

Comment:
lets make it so sitrep should not fail early - eg we want to know the entire state of the system, then if any parts of the overall command fail, we should exit with non-zero exit code.